ubuntu-headless-qt/README.md

```diff
@@ -0,0 +1,996 @@
+# RZG2L SBC board #
+This is the quick startup guide for RZG2L SBC board (hereinafter referred to as `RZG2L-SBC`).
+The below will describe the current status of development, how to build, set up environment for RZG2L-SBC.
+
+## Status
+This is a Ubuntu release of the RZG2L development product for RZG2L-SBC.
+
+This release provides the following features:
+
+ - Ubuntu build compatible with RZG2L SoC
```

Comment:
What is "Ubuntu build compatible with RZG2L SoC"?

In case of Yocto project, we have all configurations for RZG2L SoC. But in this case, what is the build system that you are using? You just create build scripts and that's all?

Reply:
I will change to :
```
 - Custom Ubuntu Core build scripts for easy setup and deployment.
```